test/JDBC/expected/TestPoint-vu-prepare.out

```diff
@@ -0,0 +1,133 @@
+CREATE TABLE POINTGEOM_dt(location GEOMETRY);
+prepst#!#INSERT INTO POINTGEOM_dt(location) values(?) #!#GEOMETRY|-|location|-|Point(47.65100 -22.34900):4326
+~~ROW COUNT: 1~~
+
+prepst#!#exec#!#GEOMETRY|-|location|-|Point(1.0 2.0):4326
+~~ROW COUNT: 1~~
+
+prepst#!#exec#!#GEOMETRY|-|location|-|Point(47.65100 -22.34900):0
+~~ROW COUNT: 1~~
+
+prepst#!#exec#!#GEOMETRY|-|location|-|Point(47.65100 -22.34900):1000000000
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Error converting data type varbinary to geometry.)~~
```

Comment:
Suggestion: We should add a comment before each command where the command is expected to throw error.

Reply:
Done